app/forms/flood_risk_engine/steps/confirmation_form.rb

```diff
@@ -19,12 +19,11 @@ def self.params_key
       property :email, virtual: true
 
       def email
-        # If we need to display multiple emails e.g contact and email_someone_else then this works
-        # emails = [enrollment.correspondence_contact.email_address]
-        # emails.map(&:downcase).uniq.to_sentence
-
-        # For now
-        enrollment.correspondence_contact.email_address
+        emails = [
+          enrollment.correspondence_contact.email_address,
+          enrollment.secondary_contact.try(:email_address)
+        ]
+        emails.delete_if(&:blank?).map(&:downcase).uniq.to_sentence
```

Comment:
I felt `delete_if(&:blank?)` was more explicit than `compact` or `reject` and handled both nil and empty strings.